src/form-builder/components/ObsControlScriptEditorModal.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
```

Comment:
I see it has a lot of code similar to ScriptEditorModal, can we bring out a common component?

Reply:
ObsControlScriptEditorModal has 2 different points from ScriptEditorModal.
1. CodeMirror property : example: readonly , autofocus 
2. componentDidUpdate to support dynamic loading of codemirror text based on obs control

We can write in a single component with some conditions, but i felt it will make one file clumsy